src/components/dialogs/advanced-search.vue

```diff
@@ -16,12 +16,13 @@
             </v-card-title>
 
             <v-card-text class="copy-normal py-4">
-              Find an existing Name Request by searching for one or more of the following criteria:
+              Find an existing Name Request by searching for <strong>one or more</strong> of the following criteria:
             </v-card-text>
 
             <!-- Validation error message -->
-            <div v-if="invalidSearch" class="error-message copy-small my-n2">
-              Enter at least one search criteria
+            <div class="my-n2">
+              <span v-if="invalidSearch" class="copy-small error-message">Enter at least one search criteria</span>
+              <br v-else> <!-- Preserve Spacing when no errors present -->
```

Comment:
Why a br instead of an empty span?

Reply:
Or you could use `&nbsp;` to actually render a space character.

Reply:
a span would require additional styling where a `br` does not.

Reply:
OK. Weird. But OK.